_lifeofinternatpuppet/post_42.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Week 42
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 42
+---
+
+Hey everyone,
+
+A reply was sent to the IT Support in the ticket raised last week after chatting to my mentor Paula in my weekly 1:1 session of what to do with the 'unapproved process page' archived in the Puppet Forge Space on Confluence.
+Check the ticket [here](https://tickets.puppetlabs.com/servicedesk/customer/portal/2/HELP-27502)
```

Comment:
```suggestion
Check the ticket [here](https://tickets.puppetlabs.com/servicedesk/customer/portal/2/HELP-27502)
```

This won't be visible publicly - I would drop this line out.

Reply:
Check the new commit message below @sanfrancrisko